### ISSUE TITLE
Visdom & Pillow-simd

### ISSUE THREAD

**Issue by 0phoff** [facebookresearch/visdom#367]
I am using pillow-simd, which is a drop-in replacement for pillow that offers a speedup for certain operations by using SIMD operations of your CPU.  
However, when installing visdom, it installs pillow, which overwrites my pillow-simd install.

They solved this in pytorch/vision, by checking if pillow-simd is installed on the system in __setup.py__.
```python
from pkg_resources import get_distribution, DistributionNotFound

def get_dist(pkgname):
    try:
        return get_distribution(pkgname)
    except DistributionNotFound:
        return None

requirements = [ ... ]
pillow_ver = ' >= 4.1.1'
pillow_req = 'pillow-simd' if get_dist('pillow-simd') is not None else 'pillow'
requirements.append(pillow_req + pillow_ver)
```

I think the only drawback of using this, is if you provide wheels, as it would then be unable to determine whether you use pillow-simd or not.  
I guess this is no concern here right?

**Comment by JackUrb** [comment]
Hi @0phoff - this kind of fix seems like a valid thing for us to do as well. I'm unfamiliar with `pillow-simd` though, would this also need to change the import statements, or is the fix to setup.py sufficient? 

Also if pillow-simd is truly a drop-in replacement with the same functionality, I don't think that it is important for visdom to know which you are using.

**Comment by 0phoff** [comment]
That is the only thing required.  
Just like Pillow and PIL, pillow-simd also uses the PIL namespace.
Thus it does indeed not matter which one someone is using.

Pillow-simd is actually a fork of Pillow and they changed some of the underlying image functions.
You can find more information in their [repository](https://github.com/uploadcare/pillow-simd).

**Comment by JackUrb** [comment]
Alright in that case I'm happy to take the pytorch/vision solution. Do you want to submit the PR?

**Comment by JackUrb** [comment]
Implemented in #368
